server/src/test/java/org/opensearch/index/store/remote/TestUtils.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.store.remote;
+
+import org.apache.lucene.tests.util.LuceneTestCase;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Comparator;
+
+import static org.junit.Assert.fail;
+
+public final class TestUtils {
```

Comment:
There's at least one other `TestUtils` in the repo. Maybe rename this something a bit more specific, like `PathUtils` or `PathTestUtils`?